curp/src/log_entry.rs

```diff
@@ -64,7 +64,13 @@ where
     pub(super) fn id(&self) -> &ProposeId {
         match self.entry_data {
             EntryData::Command(ref cmd) => cmd.id(),
-            EntryData::ConfChange(ref id) | EntryData::Shutdown(ref id) => id,
+            EntryData::ConfChange(ref id) => id,
```

Comment:
Will the id of ConfChange also be removed in subsequent PRs?

Reply:
No

Reply:
Because `ConfChange` need to store it result to `cmd_board`, and `cmd_board` use `ProposeId` as index